front/app/containers/Authentication/steps/EmaillessSsoEmail/index.tsx

```diff
@@ -37,7 +37,7 @@ interface Props {
   }) => Promise<void>;
 }
 
-const ClaveUnicaEmail = ({ loading, setError, onSubmit }: Props) => {
+const EmaillessSsoEmail = ({ loading, setError, onSubmit }: Props) => {
```

Comment:
Don't know if this name `EmaillessSsoEmail` needs some explanation as on first glance it doesn't make much sense

Reply:
Added a comment. I hope it makes it clearer.